patch-capfram.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+cd node_modules/capital-framework/src/cf-core/src
+sed -i '' -e 's|../../|~|g' cf-core.less
```

Comment:
Sounds like these are things we should rethink in CF. It looks like both instances are paths to the normalize files, are there others I'm missing?